README.md

```diff
@@ -1,89 +1,63 @@
 
-<img src="./img/logo.png" width="100">
-
 Kubeapps is a set of tools written by [Bitnami](https://bitnami.com) to super-charge your Kubernetes cluster with:
- * Your own applications [dashboard](https://kubeapps.com/), allowing you to deploy Kubernetes-ready applications into your cluster with a single click.
- * [Kubeless](http://kubeless.io/) - a Kubernetes-native Serverless Framework, compatible with [serverless.com](https://serverless.com).
- * [SealedSecrets](https://github.com/bitnami/sealed-secrets) - Encrypt your Secret into a SealedSecret, which is safe to store - even to a public repository. 
-
-These tools are easily deployed into your cluster with just one command: ```kubeapps up``` 
-
-----
-
-## Getting started
+ * Your own application [dashboard](https://kubeapps.com/), allowing you to deploy Kubernetes-ready applications into your cluster with a single click.
+ * [Kubeless](http://kubeless.io/), a Kubernetes-native Serverless Framework, compatible with [serverless.com](https://serverless.com).
+ * [SealedSecrets](https://github.com/bitnami/sealed-secrets), a way to encrypt a Secret into a SealedSecret, which is safe to store...even for a public repository.
 
-To get started, read our [Getting Started guide](docs/getting-started.md).
+## Quickstart
 
-You can also browse the [full documentation](docs/).
+Kubeapps assumes a working Kubernetes (v1.7+) with RBAC enabled and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and configured to talk to your Kubernetes cluster. Kubeapps binaries are available for both Linux and OS X, and Kubeapps has been tested with both `minikube` and Google Kubernetes Engine (GKE).
 
-----
-## Build the Kubeapps installer from source
+> On GKE, you must either be an "Owner" or have the "Container Engine Admin" role in order to install Kubeapps.
 
-The Kubeapps Installer is a CLI tool written in Go that will deploy all the Kubeapps components into your cluster.
-You can build the latest Kubeapps Installer from source. 
+The simplest way to try Kubeapps is to deploy it with the Kubeapps Installer on [minikube](https://github.com/kubernetes/minikube). 
 
-### Installing Go
-- Visit https://golang.org/dl/
-- Download the most recent Go version (here we used 1.9) and unpack the file
-- Check the installation process on https://golang.org/doc/install
-- Set the Go environment variables
+On Linux:
 
 ```
-export GOROOT=/GoDir/go
-export GOPATH=/GoDir/go
-export PATH=$GOPATH/bin:$PATH
-```
-
-### Create a working directory for the project
-
-```
-working_dir=$GOPATH/src/github.com/kubeapps/
-mkdir -p $working_dir
-```
-
-### Clone kubeapps/kubeapps repository
-
-```
-cd $working_dir
-git clone https://github.com/kubeapps/kubeapps
+minikube start
+sudo curl -s https://api.github.com/repos/kubeapps/kubeapps/releases/latest | grep linux | grep browser_download_url | cut -d '"' -f 4 | wget -i -
```

Comment:
I'm using this so the user always gets the latest release, rather than having the command hardcoded to a specific version (so there's less need to update it every time). I got the command from https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8 and tested it, seems to work fine but please also test it yourself.

Reply:
Well, this is about building the latest source. For documentation on how to use, we have the rest of the documents. Please, leave here the documentation on how to build. Also, please, avoid having explicit reference to minikube on the docs.

Reply:
Added back in README, removed minikube commands